triggers.cwt

```diff
@@ -3483,4 +3484,47 @@ alias[trigger:hidden_tooltip] = {
 
 alias[trigger:show_scope_change] = no
 alias[trigger:generate_tooltip] = no
-alias[trigger:show_only_failed_conditions] = yes
\ No newline at end of file
+alias[trigger:show_only_failed_conditions] = yes
+
+# TODO: Doc
+## scope = siege
+### 
+alias[trigger:phase] = enum[combat_phases]
+
+# TODO: Doc
+## scope = siege
+### 
+alias[trigger:enum[troop_types_land_only]] = float
+
+# TODO: Doc
+## scope = siege
+### 
+alias[trigger:<special_troop_type>] = float
+
+# TODO: Doc
+## scope = siege
+### 
+alias[trigger:days] = int
```

Comment:
Do we think these are triggers that work everywhere (in siege scope)?
They're not combat_tactics specific somehow right?

Reply:
I don't particularly know, I haven't seen siege scope being used much outside of combat_tactics

Reply:
Should I put this in issues?

Reply:
Nah, I've had a look at vanilla and they look like real triggers, so it's fine :)